windows_containers/windows-containers-release-notes-6-x.adoc

```diff
@@ -33,22 +33,33 @@ endif::openshift-origin[]
 
 This release of the WMCO provides bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 6.0.0 were released in
 
+[id="wmco-6.0.0-new-features"]
 === New features and improvements
+
 [id="wmco-6.0.0-node-certificates"]
 ==== Windows node certificates are updated
 
 With this release, the WMCO updates the Windows node certificates when the kubelet client certificate authority (CA) certificate is rotated.
 
-[id="wmco-6-0-0-new-features"]
-=== New features
+[id="wmco-6.0.0-systemd"]
+==== The containerd runtime is now being used with Windows nodes 
+
+Previously, the Windows nodes used a Docker-formatted container runtime. To align with the upstream Kubernetes deprecation of Docker as a container runtime, containerd 1.6.0 is the new supported container runtime for Windows nodes. 
 
-[id="wmco-6-0-0-containerd"]
-==== Containerd is the default container runtime
+Windows instances deployed by the WMCO are configured with the containerd container runtime. Because the WMCO installs and manages the runtime, it is recommended that you do not manually install containerd on nodes.
 
-Because the Docker runtime is deprecated in Kubernetes 1.24, containerD is now the default runtime for WMCO-supported Windows nodes. Upon the installation of or an upgrade to WMCO 6.0.0, containerd is installed as a Windows service. The kubelet now uses containerd for image pulls instead of the Docker runtime. Users no longer need to enable the Docker-formatted container runtime or install the Docker container runtime on Bring-Your-Own-Host (BYOH) instances. You can continue to use nodes based on VM images that use Docker. containerd can run along with the Docker service.
+The kubelet uses the containerd runtime for image pulls instead of the Docker runtime. Users no longer need to enable the Docker-formatted container runtime or need to install the Docker container runtime on Bring-Your-Own-Host (BYOH) Windows instances. 
 
 The WMCO supports a Windows golden image with or without Docker for vSphere and Bring-Your-Own-Host (BYOH) Windows instances. 
 
+After updating to {product-title} {product-version}, the kubelet no longer uses the Docker runtime. However, you can continue to use Docker services on your nodes, as needed, for non-kubelet-related purposes. Containerd can run alongside the Docker service.
+
+[id="wmco-6.0.0-known-issues"]
+=== Known issues
+
+//https://issues.redhat.com/browse/WINC-846
+* Because of a containerd configuration issue, for Windows nodes managed by the WMCO, you cannot pull container images from private registries. It is recommended that you use images from public registries or pre-pull the images.
```

Comment:
This issue has been prior to containerd and is not result of a configuration issue, added some clarification the PR: 
https://github.com/openshift/windows-machine-config-operator/pull/1174#discussion_r946780208